servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/OutlierDetectorConfig.java

```diff
@@ -293,13 +575,29 @@ public Builder consecutive5xx(final int consecutive5xx) {
 
         /**
          * Set the interval on which to run failure percentage and success rate failure detectors.
-         * Defaults to 10 seconds.
+         * This method will also use either the default jitter or the provided interval, whichever is smaller.
+         * Defaults to 10 second interval with 3 second jitter.
          * @param interval the interval on which to run failure percentage and success rate failure detectors.
          * @return {@code this}
          */
         public Builder interval(final Duration interval) {
```

Comment:
Consider using a self-explanatory name, just "interval" doesn't tell what is this interval for.

Reply:
I changed it to `scanInterval`. Let me know if that is a bit more self explanatory.

Reply:
Continuation: https://github.com/apple/servicetalk/pull/2864#discussion_r1539742231